MinVer/MinVer.targets

```diff
@@ -21,12 +21,16 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(DesignTimeBuild)' != 'true'">
+  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(UsingMicrosoftNETSdk)' == 'true' And '$(DesignTimeBuild)' != 'true'">

```

Comment:
There's no harm in having this target run, so I'm not sure the extra condition is necessary.

Reply:
> There's no harm in having this target run, so I'm not sure the extra condition is necessary.

I did consider that, but I'm not really it's good to have "half" of MinVer working. I guess the NuGet package might still get the right version number? But anyway, it's untested, and I'm not sure I want to deal with bugs raised around usage in legacy csproj. I think it's OK to support SDK projects only and prevent anything from happening in legacy csproj.

> If DependsOnTargets are evaluated first, then someone without the .NET Core SDK installed using a legacy project could get an error from the Exec task before seeing the warning.

I don't think this problem exists. I've tested what happens in a legacy project and I can confirm that `MinVer_Warnings` is run, but neither `MinVer` nor `MinVer_GetVersion` are run. That kind of makes sense to me, since the `BeforeTargets` is saying _just_ that, i.e. run _before_ that target is run. `DependsOnTargets` is saying, _if this target is run_, run this other target first.

Reply:
Oh wait, I see what you're saying. If the condition was taken out, would it be a problem? I guess it would be, since we're not checking whether they have the SDK installed.

Reply:
But what I said still holds, `MinVer_Warnings` runs before `MinVer_GetVersion`, so the exact problem you describe wouldn't happen.

Reply:
@bording I'm about to release alpha 9. Do you have any more thoughts on this, or do you think it's good to go? There's no rush, it can wait for alpha 10.

Reply:
I'd like to validate a couple of things first, so alpha 10 sounds ok to me.

Reply:
After looking at this, I'm now convinced that adding the extra condition is the right call since it prevents any problems with not having the .NET Core SDK installed.